clients/src/main/resources/common/message/BrokerRegistrationRequest.json

```diff
@@ -50,6 +50,8 @@
     ]
     },
     { "name": "Rack", "type": "string", "versions": "0+", "nullableVersions": "0+",
-      "about": "The rack which this broker is in." }
+      "about": "The rack which this broker is in." },
+    { "name": "ZkMigrationReady", "type": "int8", "versions": "0+", "taggedVersions": "0+", "tag": 0, "ignorable": true,
```

Comment:
It would be nice if this name matched up with the field in the broker registration.

Maybe something like `IsMigratingZkBroker` ?